modules/core/04-channel/keeper/timeout.go

```diff
@@ -80,6 +80,10 @@ func (k Keeper) TimeoutPacket(
 
 	commitment := k.GetPacketCommitment(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
 
+	if len(commitment) == 0 {
+		return sdkerrors.Wrapf(types.ErrPacketTimedOut, "packet sequence (%d)", packet.GetSequence())
+	}
```

Comment:
ditto. Also how do you know  if a packet commitment is deleted because it timed out or because it was acknowledged?